lib/memo_wise.rb

```diff
@@ -45,4 +47,21 @@ def #{method_name}(*args)
       end
     end
   end
+
+  def reset_memo_wise(method_name)
+    unless respond_to?(method_name)
```

Comment:
Should we similarly enforce that this is a symbol? Otherwise I think this might not clear correctly if we pass in a string.